modules/openapi-generator/src/main/resources/cpp-ue4/model-source.mustache

```diff
@@ -175,9 +175,19 @@ bool {{classname}}::FromJson(const TSharedPtr<FJsonValue>& JsonValue)
 
 	bool ParseSuccess = true;
 
-    {{#vars}}
+	{{#vars}}
+	{{#required}}
+	{{#isWriteOnly}}
+	TryGetJsonValue(*Object, TEXT("{{baseName}}"), {{name}});
+	{{/isWriteOnly}}
+	{{^isWriteOnly}}
```

Comment:
```suggestion
	{{^isWriteOnly}} ParseSuccess &= {{/isWriteOnly}} TryGetJsonValue(*Object, TEXT("{{baseName}}"), {{name}});
```

Slightly more concise version, otherwise the change looks great.

Reply:
change made locally, tested and pushed.